blocked-edges/4.18.0-MCORestoreCon.yaml

```diff
@@ -0,0 +1,9 @@
+to: 4.18.0
+from: 4[.]17[.].*
+fixedIn: 4.18.13
+url: https://issues.redhat.com//browse/OCPBUGS-54765
```

Comment:
This can point at https://issues.redhat.com/browse/OCPNODE-3245 , which is getting set up with a customer-facing description focused on helping cluster-admins weigh the risk.  The bug is focused on helping developers and QE build and test a fix, so it has content that's not immediately relevant to update-risk-weighing.

Reply:
done!